src/main/java/org/prebid/server/bidder/adquery/model/request/AdQueryRequest.java

```diff
@@ -0,0 +1,57 @@
+package org.prebid.server.bidder.adquery.model.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@Builder(toBuilder = true)
+@EqualsAndHashCode
+@Getter
+public class AdQueryRequest {
+
+    String v;
+
+    @JsonProperty("placementCode")
+    String placementCode;
+
+    @JsonProperty("auctionId")
+    String auctionId;
+
+    String type;
+
+    @JsonProperty("AdUnitCode")
```

Comment:
a typo `adUnitCode`

P.S. please double-check the contract once more time for all the DTOs